options.js

```diff
@@ -18,6 +18,7 @@ var git = {
   
   var options = (function () {
     var options = {
+      host: git.config.get('gitlab.url') || process.env.GITLAB_URL,
```

Comment:
I am not sure if this is a correct change. Can you let me know what your `gitlab.url` value is. Using it in host and url both doesnt make sense.

Reply:
> I am not sure if this is a correct change. Can you let me know what your `gitlab.url` value is. Using it in host and url both doesnt make sense.

I set the global environment variables `GITLAB_URL=https://code.choerodon.com.cn`

![image](https://github.com/vishwanatharondekar/gitlab-cli/assets/28728567/82ec044a-709d-4802-bc44-dba4cd46b71d)

Reply:
> I am not sure if this is a correct change. Can you let me know what your `gitlab.url` value is. Using it in host and url both doesnt make sense.

The host would be set to https://gitlab.com without host options in @gitbeaker\requester-utils\dist\index.js. I guess the incorrect host leading to 401 response.

var _b = _a === void 0 ? {} : _a, token = _b.token, jobToken = _b.jobToken, oauthToken = _b.oauthToken, sudo = _b.sudo, profileToken = _b.profileToken, requesterFn = _b.requesterFn, _c = _b.profileMode, profileMode = _c === void 0 ? 'execution' : _c, **_d = _b.host, host = _d === void 0 ? 'https://gitlab.com' : _d**

Reply:
I checked this and realise that you are correct. This happened due to moving to a new library in 2.0.10 which requires host instead of url. 

Can you do the change of removing url since thats not required anymore.

Reply:
Great work @Holinc19 on debugging and finding out what the real issue is. 👍